src/explore-education-statistics-admin/src/components/comments/__tests__/Comment.test.tsx

```diff
@@ -0,0 +1,126 @@
+import Comment from '@admin/components/comments/Comment';
+import {
+  testComments,
+  testCommentUser1,
+  testCommentUser2,
+} from '@admin/components/comments/__data__/testComments';
+import { AuthContext, User } from '@admin/contexts/AuthContext';
+import { GlobalPermissions } from '@admin/services/permissionService';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+describe('Comment', () => {
+  const testUser1: User = {
+    id: testCommentUser1.id,
+    name: `testCommentUser1.firstName testCommentUser1.lastName`,
```

Comment:
Just wondering if you meant to wrap this in a dollar sign & curly brackets? i.e. `${testCommentUser1.firstName} ${testCommentUser1.lastName}`

Reply:
Doh! Yes, sorted now.